### PR TITLE
Add CSV export for issues and discussions, refs #23

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: udata/core/organization/views.py [hunk 1]
```diff
@@ -291,6 +296,26 @@ def datasets_csv(org):
     return csv.stream(adapter, '{0}-datasets'.format(org.slug))
 
 
+@blueprint.route('/<org:org>/issues.csv')
+def issues_csv(org):
+    datasets = Dataset.objects.filter(organization=str(org.id))
+    issues = [DatasetIssue.objects.filter(subject=dataset)
```
Comment (noirbizarre) [inline udata/core/organization/views.py]: I think ``DatasetIssue.objects(subject__in=datasets)`` should be more efficient

File: udata/core/organization/views.py [hunk 2]
```diff
@@ -291,6 +296,26 @@ def datasets_csv(org):
...
+              for dataset in datasets]
+    # Turns a list of lists into a flat list.
+    adapter = IssuesOrDiscussionCsvAdapter(itertools.chain(*issues))
+    return csv.stream(adapter, '{0}-issues'.format(org.slug))
+
+
+@blueprint.route('/<org:org>/discussions.csv')
+def discussions_csv(org):
+    datasets = Dataset.objects.filter(organization=str(org.id))
+    discussions = [DatasetDiscussion.objects.filter(subject=dataset)
```
Comment (noirbizarre) [inline udata/core/organization/views.py]: I think ``DatasetDiscussion.objects(subject__in=datasets)`` should be more efficient

File: udata/core/organization/views.py (hunk 1 above)
Comment (davidbgk) [inline udata/core/organization/views.py]: Indeed! Thanks.

Comment (davidbgk) [inline udata/core/organization/views.py]: After trying a few things, it appears that the planned `ReferenceField` refactoring is necessary to get rid of the manual list comprehension. I'll keep it as-is for now.

**Pull request merged**